js/src/ScatterBase.ts

```diff
@@ -638,8 +647,8 @@ export abstract class ScatterBase extends Mark {
         this.send({
             event: "drag_end",
             point: {
-                x: x_scale.invert(d[0]),
-                y: y_scale.invert(d[1])
+                x: (x_scale as LinearScale | OrdinalScale).invert(d[0]),
+                y: (y_scale as LinearScale | OrdinalScale).invert(d[1])
```

Comment:
I think there is a design issue here. There is a confusion between the `x_scale` attribute and the local `x_scale` variable in the code.

I am setting the `x_scale` to `Scale` attribute instead of `any` for now, in order to keep the code similar to the older implementation. We might want to address this issue at some point and enforce the `x_scale` attribute to `LinearScale | OrdinalScale` in `Scatter`.

The same comment for `y_scale` obviously.